### PR TITLE
This commit reflects changes for PLANET-3641

### PR DESCRIPTION
**Pull request by Osong-Michael**
### Reference PR: [976](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/976)

- There changes in this commit are more indentation fixes. Only spacing and indentation issues were fixed.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [phoebe](https://www-dev.greenpeace.org/test-phoebe) | [admin](https://www-dev.greenpeace.org/test-phoebe/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-phoebe/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-phoebe) | [composer-local.json](https://github.com/greenpeace/planet4-test-phoebe/blob/main/composer-local.json)

:watch: 2023.02.05 14:59:15

**Review by mleray — COMMENTED** [review]
Could you please change the commit title to something a bit more clear, making it about the indentation fix?
Also, I notice big changes in the `package-lock.json` file, is that intended? 🤔

**Comment by Osong-Michael** [comment]
> Could you please change the commit title to something a bit more clear, making it about the indentation fix? Also, I notice big changes in the `package-lock.json` file, is that intended? 🤔

I am sure the changes to the `package-lock.json` are due to the fact that I upgraded my node version to 15 and had to reinstall all packages. As for the commit title, let me change that now.

File: assets/src/components/NavigationType/NavigationType.js [hunk 1]
```diff
@@ -10,12 +10,12 @@ const NAVIGATION_TYPE_MINIMAL = 'minimal';
 export const NavigationType = ({value, setValue, defaultValue, options}) => {
   const defaultOptions = [
     {
-      "label": __( 'Main website navigation', 'planet4-blocks-backend' ),
-      "value": NAVIGATION_TYPE_PLANET4,
+      'label': __( 'Main website navigation', 'planet4-blocks-backend' ),
```
Comment (mleray) [inline assets/src/components/NavigationType/NavigationType.js:13]: Can't we just remove the quotes entirely in this case? 🤔

File: assets/src/components/Sidebar/ThemeSettings.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ export const ThemeSettings = props => {
   const { editPost } = useDispatch('core/editor');
 
   useEffect(() => {
-    handleThemeSwitch('theme', meta.theme, meta)
+    handleThemeSwitch('theme', meta.theme, meta);
```
Comment (mleray) [inline assets/src/components/Sidebar/ThemeSettings.js:58]: Same comment here as earlier, not a blocker but we can probably shorten this as follows:
```
useEffect(() => handleThemeSwitch('theme', meta.theme, meta), [meta.theme]);
```

**Comment by mleray** [comment]
> I am sure the changes to the `package-lock.json` are due to the fact that I upgraded my node version to 15 and had to reinstall all packages.

That might be, but I know I've had issues with the lock file in the past too (in fact recently when adding Playwright) and the solution was to re-generate it using `make deps`

File: assets/src/components/ShareButtons/ShareButtons.js [hunk 3]
```diff
@@ -17,57 +17,57 @@ export const ShareButtons = ({social_params, social_accounts}) => {
...
-  }
+      eventLabel: label,
+    });
+  };
 
   return (
     <div className="share-buttons">
       <a href={ `https://wa.me/?text=${encodeURIComponent(link)}&${utm('whatsapp', utm_medium, utm_content, utm_campaign)}` }
-        onClick={() => {share('Whatsapp', link)}}
+        onClick={() => {share('Whatsapp', link);}}
```
[the remark this answers is not in the document]
Comment (Osong-Michael) [reply assets/src/components/ShareButtons/ShareButtons.js]: Fixed

File: assets/src/components/Sidebar/ThemeSettings.js (hunk 2 above)
Comment (Osong-Michael) [reply assets/src/components/Sidebar/ThemeSettings.js]: Fixed

**Comment by comzeradd** [comment]
> I am sure the changes to the package-lock.json are due to the fact that I upgraded my node version to 15 and had to reinstall all packages.

We should clean that up before merging though since it will cause issues on other people local development environemnt. In thoery, runnng `make deps` should be fine since docker-compose [uses node:14](https://github.com/greenpeace/planet4-docker-compose/blob/main/docker-compose.yml#L113).

File: assets/src/components/ShareButtons/ShareButtons.js [hunk 4]
```diff
@@ -17,57 +17,61 @@ export const ShareButtons = ({social_params, social_accounts}) => {
...
+        rel="noreferrer"
       >
-        <SvgIcon {...{name: "whatsapp"}} />
+        <SvgIcon {...{name: 'whatsapp'}} />
         <span className="visually-hidden">{__( 'Share on', 'planet4-blocks' )} Whatsapp</span>
       </a>
 
       <a href={ `https://www.facebook.com/sharer/sharer.php?u=${encodeURIComponent(link)}&${utm('facebook', utm_medium, utm_content, utm_campaign)}` }
-        onClick={() => {share('Facebook', link)}}
+        onClick={() => {share('Facebook', link);}}
```
Comment (mleray) [inline assets/src/components/ShareButtons/ShareButtons.js:37]: You can make the same change here as you did for [line 27](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1002/files#diff-2e64eedbd1a146969a412a57337e18b1dd1d738e5fc673387bbeabefbf9299a9R27), also for the 2 following `onClick` functions in the file

Comment (Osong-Michael) [reply assets/src/components/ShareButtons/ShareButtons.js]: Fixed

File: assets/src/components/NavigationType/NavigationType.js (hunk 1 above)
Comment (Osong-Michael) [reply assets/src/components/NavigationType/NavigationType.js:13]: Made this changes and also rebased my branch to use the latest changes in main and node version too. I will also update some fixes in other branches I am currently working on.

**Review by mleray — CHANGES_REQUESTED** [review]
I see some submodule changes that I don't think are intended 🤔 Also the `package-lock.json` file issue is still there!

File: assets/src/components/ShareButtons/ShareButtons.js (hunk 4 above)
Comment (mleray) [reply assets/src/components/ShareButtons/ShareButtons.js]: I think that you didn't update the following `onClick` functions? There are 2 more in the file

**Comment by GP-Dan-Tovbein** [comment]
> > Could you please change the commit title to something a bit more clear, making it about the indentation fix? Also, I notice big changes in the `package-lock.json` file, is that intended? 🤔
> 
> I am sure the changes to the `package-lock.json` are due to the fact that I upgraded my node version to 15 and had to reinstall all packages. As for the commit title, let me change that now.

For now, I think we should use `NodeJS 14` for now since it is the version we currently use. So, maybe all deps from the `package-lock` should have all its valid versions.

**Pull request closed without merging**